test/karma.config.ts

```diff
@@ -89,7 +101,7 @@ export function config(config) {
     browserNoActivityTimeout: 100000,
     browsers: ['Chrome'],
 
-    singleRun: false
+    singleRun: true
```

Comment:
Process never gets past the tests and thus won't report coverage.

IMO it should really be split into test & test-watch